### ISSUE TITLE
Feature Request: Support for a rake deploy task?

### ISSUE THREAD

**Issue by wakatara** [masasam/emacs-easy-jekyll#3]
**TLDR**

Can you support arbitrary rake tasks for a deploy command (or any other configurable shell command)?

**The Longer Tale**

I've used s3_website to configure both Amazon S3 and Cloudfront to deliver a high performance and very fast site for my blog. I also strip all the .html off of most files in order to have nice clean URLs before sending to S3 via s3_website push (it actually kinda rocks and love the Cloudfront chacing).

I do this via a `rake deploy` task which I'm still doing manually as it's unclear to me from the docs if you can support that in easy-jekyll. 

Is there a mechanism to do this via easy-jekyll (either confif or alteration) and, if not, could you add executing rake tasks or similar shell scripts as a deploy option?

thanks! I really like how put together this is.  =] 

I'm a recent emacs convert, so was very happy to find easy-jekyll. My

**Comment by masasam** [comment]
Hi @wakatara .
Thanks for comment.

You can do it in the following way.
put deploy.sh and Rakefile at easy-jekyll-basedir

deploy.sh

    #!/bin/sh
    rake hello
    rake pen
    rake apple

chmod a+x deploy.sh

Rakefile

    task :hello do
      File.open("hello.txt", "w") do |f|
        f.puts("Hello, World!")
      end
    end
    task :pen do
      File.open("pen.txt", "w") do |f|
        f.puts("I have a pen!")
      end
    end
    task :apple do
      File.open("apple.txt", "w") do |f|
        f.puts("I have an apple!")
      end
    end

M-x easy-jekyll-github-deploy
or
Press G on easy-jekyll-mode

**Comment by wakatara** [comment]
Perfect. Looks easy peasy. Thanks! Loving the mode.  :-)
